src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py

```diff
@@ -157,12 +232,22 @@ def connect(auth):
 
 @socketio.on('disconnect', namespace='/results')
```

Comment:
Maybe the function @socketio.on('disconnect', namespace='/data') is missing?

Reply:
done